spec/tapioca/cli/gem_spec.rb

```diff
@@ -278,9 +278,10 @@ class Foo::Secret; end
           File.delete(repo_path / "sorbet/tapioca/require.rb")
         end
       end
+
       describe("sync") do
         it "must perform no operations if everything is up-to-date" do
-          execute("generate")
+          execute("gem")
```

Comment:
This was previously using the deprecated `generate` command, which I'm guessing was due to copy/pasta? Regardless, I updated to use the `gem` command and tests are passing as expected.